src/bcf/record.rs

```diff
@@ -1483,6 +1483,9 @@ impl<'a, 'b, B: BorrowMut<Buffer> + Borrow<Buffer> + 'b> Format<'a, B> {
     /// memory.
     pub fn string(mut self) -> Result<BufferBacked<'b, Vec<&'b [u8]>, B>> {
         self.data(htslib::BCF_HT_STR).map(|ret| {
+            if ret == 0 {
```

Comment:
I'm not sure if this is the behavior we want here. Could consider returning "." but then I guess we'd need to modify the buffer

Reply:
empty seems fine